aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1658,4 +1658,83 @@ Tensor& diag_cpu_out(Tensor &result, const Tensor& self, int64_t dimension) {
   return result;
 }
 
+Tensor moveaxis(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
```

Comment:
Better make this "movedim", per @zou3519's point.

Reply:
Sure will do. 

I just feel we should alias it soon as I generally tend to search for functions with numpy names i.e. `moveaxis`, etc